hub.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: page_md
+title: JupyterHub
+title_image: assets/hublogo.svg
+tagline: A multi-user version of the notebook designed for companies, classrooms and research labs
+navbar_jupytercon: true
+cfp: true
+permalink: /hub
+---
+
+## What is JupyterHub?
+
+JupyterHub provides remote access to Jupyter servers on shared infrastructure
+in ordre to make computational environments and resources more accessible to
+students, researchers, and data scientists.
+
+JupyterHub runs in the cloud or on your own hardware, and makes it possible
+to serve a pre-configured data science environment to any user in the world.
+It is customizable and scalable, and is suitable for small and large teams,
+academic courses, and large-scale infrastructure.
+
+## Key features of JupyterHub
+
+**Flexible** - JupyterHub can be used to serve a variety of environments. It
+supports dozens of kernels with the Jupyter server, and can be used to serve
+a variety of user interfaces including the Jupyter Notebook, Jupyter Lab,
+RStudio, nteract, and more.
+
+**Secure** - JupyterHub can be configured with authentication in order to
```

Comment:
do you mean to combine scalable and flexible into a single "flexible" section?

Reply:
I think that we should remove secure as an element and instead call the authentication "flexible" to avoid implicit guarantees of security levels esp. on use of authenticators not maintained by JupyterHub.

Reply:
Renaming the Flexible section above to another synonym meaning flexible.

Reply:
hehe, ok :-) 

how about "customizable"?

Reply:
That works.